src/App.test.js

```diff
@@ -715,4 +716,50 @@ describe('App component', () => {
       expect(localStorageItems['dopplerFirstOrigin.value']).toBeUndefined();
     });
   });
+
+  describe('activation in progress parameter', () => {
+    it('should call google adwords', () => {
```

Comment:
Could you include the condition?

```suggestion
    it('should call google adwords when query string contains `activationInProgress%20=%20true`', () => {
```

Reply:
changed